packages/@aws-cdk/aws-appmesh/test/test.virtual-router.ts

```diff
@@ -353,6 +369,9 @@ export = {
               weight: 50,
             },
           ],
+          timeout: {
+            idle: cdk.Duration.seconds(10),
+          },
```

Comment:
If we're not going to change the assertions in this file, please don't see the timeout values (they don't add anything to these tests).

Reply:
yes missed this one, adding the assertion.